app/src/main/java/com/sun/noteapp/ui/home/MainActivity.kt

```diff
@@ -0,0 +1,55 @@
+package com.sun.noteapp.ui.home
+
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.Menu
+import androidx.appcompat.app.ActionBarDrawerToggle
+import androidx.appcompat.widget.Toolbar
+import androidx.core.view.GravityCompat
+import androidx.drawerlayout.widget.DrawerLayout
+import com.google.android.material.navigation.NavigationView
+import com.sun.noteapp.R
+
+class MainActivity : AppCompatActivity() {
+    private lateinit var drawer: DrawerLayout
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        initView()
+    }
+
+    private fun initView() {
+        val toolbar = findViewById<Toolbar>(R.id.toolbar_home)
+        setSupportActionBar(toolbar)
```

Comment:
tại sao anh không viết như này 
```suggestion
        setSupportActionBar(toolbar_home)
```